docs/helm/endpoints_api.md

```diff
@@ -334,6 +334,9 @@ _Returns repository index file containing all entries from all configured reposi
 * **Method:**
 
   `GET`
+  
+* **Supported URL Query Parameter:**
+    *  `onlyCompatible` - `true`/`false` Setting true would return helm charts which are supported in the provided cluster.
```

Comment:
we should state what is the default value if omitted.

Reply:
done